build.gradle

```diff
@@ -1,21 +1,23 @@
 buildscript {
 	repositories {
 		jcenter()
+		maven {
+            url "https://plugins.gradle.org/m2/"
+		}
 	}
 	dependencies {
 		classpath 'org.xtext:xtext-gradle-plugin:1.0.15'
-		classpath 'com.moowork.gradle:gradle-node-plugin:0.13'
+		classpath 'com.moowork.gradle:gradle-node-plugin:1.2.0'
```

Comment:
That looks a bit strange. Why was it there in the first place? It's using the old plugin mechanism (necessary sometimes, but deprecated).

Reply:
no idea it was no longer working